lib/cylc/scheduler.py

```diff
@@ -1134,6 +1138,21 @@ def load_suiterc(self, reconfigure):
             is_restart=self.is_restart, is_reload=reconfigure,
             mem_log_func=self.log_memory
         )
+
+        # if reconfigure, write out to the log(s) any added / removed tasks
+        if old_config:
+            add = set(self.config.get_task_name_list()) - set(
+                old_config.get_task_name_list())
+            rem = set(old_config.get_task_name_list()) - set(
+                self.config.get_task_name_list())
+            if len(add) + len(rem) > 0:
+                msgs = ["Added task: '%s'" % task for task in add]
+                msgs += ["Removed task: '%s'" % task for task in rem]
+                for msg in msgs:
+                    logging.getLogger("main").log(logging.INFO, msg)
+                    print msg
+                    print >> sys.stderr, (msg)
```

Comment:
IMO existing task proxies orphaned by removal of their definitions warrant a logged warning, which gets printed to stderr as well by default. 

New task definitions added probably also warrants a warning, as a reminder that the user has to manually insert new instances if they are to be used.

In which case, no additional printing is needed.